test/e2e-olm/utils.go

```diff
@@ -35,27 +37,48 @@ const (
 	exampleSecretsPath   = "/tmp/example-secrets"
 )
 
-func registerSchemes(t *testing.T) {
-	elasticsearchList := &loggingv1.ElasticsearchList{}
-	err := test.AddToFrameworkScheme(loggingv1.SchemeBuilder.AddToScheme, elasticsearchList)
+func setupK8sClient(t *testing.T) {
+	ok := false
```

Comment:
No need for default initialization, L42 will solve this for you, golang ensure all variables will be default initialized if not set by a user.

Reply:
Fixed. This explicit declaration is used because of the global variable in L42, which could be shadowed by default initialization.